mktrayicon.c

```diff
@@ -214,28 +211,57 @@ static GtkStatusIcon *create_tray_icon(char *start_icon)
 int main(int argc, char **argv)
 {
 	char *start_icon = "none";
+	char *tooltip = NULL;
+	char *pipe = NULL;
 	FILE *fifo;
 	GThread *reader;
 
 	XInitThreads(); /* see http://stackoverflow.com/a/18690540/472927 */
 	gtk_init(&argc, &argv);
 
-	if (argc == 4 && strcmp(argv[1], "-i") == 0) {
-		start_icon = argv[2];
-	}
-
 	if (argc == 1) {
-		printf("Usage: %s [-i ICON] FIFO\n", *argv);
-		printf("Listen to FIFO for system tray icon specifications\n");
+		printf("Usage: %s [-i ICON] [-t TOOLTIP] [FIFO]\n", *argv);
+		printf("Create a system tray icon as specified\n");
 		printf("\n");
 		printf("  -i ICON\tUse the specified ICON when initializing\n");
+		printf("  -t TOOLTIP\tUse the specified TOOLTIP when initializing\n");
+		printf("  if a FIFO is provided, it should be the last argument\n");
+		printf("  if a FIFO is not provided, mktrayicon will run until killed\n");
```

Comment:
I think I'd prefer for these lines to not be inline with the argument list, but rather after the `printf("\n")` below, without the indentation, with a capital first letter.

Reply:
Ok.